src/test/java/emissary/config/ConfigUtilTest.java

```diff
@@ -529,6 +529,7 @@ public void testMultipleMasterClassNamesMultipleDirs() throws IOException, Emiss
     }
 
     @Test
+    @Ignore
```

Comment:
Do we really want to ignore this test?

Reply:
Yes, until we address the ```LogbackCapture``` Java 11 compatibility issue.

Reply:
Fixed this separately in #201 - see https://github.com/NationalSecurityAgency/emissary/pull/201/files#diff-49ba8d133940994a3480f8831e57e54e2dc7d1f43bd3258d98cd1ff5722fccfa and file beneath for current method.